src/navigation/BottomTabs/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {PrimaryStackNavigator} from '../PrimaryStack';
+import {BottomTabRoute} from './BottomTabNavigator.types';
+import {CustomIcon, CustomIconNames} from '../../components';
+import ProfileScreen from '../../screens/ProfileScreen/ProfileScreen';
+
+const BottomTabs = createBottomTabNavigator();
+
+const BottomTabNavigator = () => {
+  const tabBarIcon = (props: {
+    focused: boolean;
+    color: string;
+    size: number;
+    icon: CustomIconNames;
+  }) => {
+    return (
+      <CustomIcon name={props.icon} size={props.size} color={props.color} />
+    );
+  };
+
+  return (
+    <BottomTabs.Navigator>
+      <BottomTabs.Screen
+        name={BottomTabRoute.Search}
+        component={PrimaryStackNavigator}
+        options={{
+          tabBarIcon: ({color, focused, size}) =>
+            tabBarIcon({
```

Comment:
Here can we do this instead - 
` tabBarIcon: (props) =>
            tabBarIcon({
              ...props,
              icon: CustomIconNames. Search,
            })`
            
            By doing this, we can pass all the props we get from the callback and only add the icon prop with our custom vaue

Reply:
Fixed.